pkg/cosign/verify.go

```diff
@@ -203,6 +214,11 @@ func Verify(ctx context.Context, ref name.Reference, co CheckOpts) ([]SignedPayl
 			}
 		}
 
+		verified, err := sp.VerifyBundle()
```

Comment:
We should explain how this works a bit in the output too, in cli/verify.go I think.

Reply:
👍🏽